### ISSUE TITLE
My fork of Dan Hamer's free chord pack

### ISSUE THREAD

**Issue by fotisandstuff** [ldrolez/free-midi-chords#3]
Hello, ldrolez! This is not really an issue, just a heads-up. Have you heard of the chord pack Dan Hamer posted on the [WATMM subreddit](https://www.reddit.com/r/WeAreTheMusicMakers/comments/lv9zq1/fed_up_of_these_1000000_midi_chord_bounty_packs/)? Well, I made an even better version of it, with additional help by Hamer himself! I fixed A LOT of filenames and folder structures. You can get it [here](https://drive.google.com/file/d/1IVuQ-1Joa0njxsRqa05dBHZfxY9obhHA/view?usp=sharing). Hope it's useful enough for your archive!

**Comment by ldrolez** [comment]
Hi!

Thanks for the link! I'll see if I can borrow a few progressions for this pack.

But are we sure that there are no usage restictions on these?

I should also start to categorize the progressions, with more than 100 chords progressions
my pack could become a mess, but I don't know how to start...

-- 
Ludo.

https://drolez.com/blog/   - Music and Tech Blog

**Comment by fotisandstuff** [comment]
Yes, Dan says on the Reddit page that they are 100% free to use and that he allows anyone to share away and host elsewhere.

**Comment by fotisandstuff** [comment]
He has a Contact page on his website, so you can always ask him: https://danielhamer.com/contact/

**Comment by fotisandstuff** [comment]
Hey! So, have you checked out my chord pack fork?

**Comment by Iamhustlerussell** [comment]
All harmonies, ever created or yet to be created, are not subject to copyright restrictions, only melodies, vocals and maybe, arguably,  the bassline... So.... These should be usage restriction free just due to the inherent nature of the progressions being harmonies...
Right?
I'm still a student in my first year of working on getting my bachelor's of science degree in music production, so I could totally be wrong, but I was just tested on this so I want to believe that I'm right?
At any rate, gonna be perusing these for inspiration soon!

**Comment by RustoMCSpit** [comment]
> All harmonies, ever created or yet to be created, are not subject to copyright restrictions, only melodies, vocals and maybe, arguably, the bassline... So.... These should be usage restriction free just due to the inherent nature of the progressions being harmonies... Right? I'm still a student in my first year of working on getting my bachelor's of science degree in music production, so I could totally be wrong, but I was just tested on this so I want to believe that I'm right? At any rate, gonna be perusing these for inspiration soon!

unless the harmony was some bizarre jazz shit thats never been done before theres no chance at copyright, you can download and share any midi files for harmony no one has the rights to it

**Comment by RustoMCSpit** [comment]
https://archive.org/details/themagicofmidiv1
check this out

**Comment by RustoMCSpit** [comment]
@ldrolez ?

**Comment by RustoMCSpit** [comment]
> Hi! Thanks for the link! I'll see if I can borrow a few progressions for this pack. But are we sure that there are no usage restictions on these? I should also start to categorize the progressions, with more than 100 chords progressions my pack could become a mess, but I don't know how to start...
> […](#)
> -- Ludo. https://drolez.com/blog/ - Music and Tech Blog

there are no restrictions to chord progressions
